src/GenAPI/Microsoft.DotNet.GenAPI/GenAPIApp.cs

```diff
@@ -198,9 +201,17 @@ private static string ReadHeaderFile(string? headerFile)
 
             """;
 
-            return !string.IsNullOrEmpty(headerFile) ?
+            string header = !string.IsNullOrEmpty(headerFile) ?
                 File.ReadAllText(headerFile) :
                 defaultFileHeader;
+
+#if NET
+            header = header.ReplaceLineEndings();
```

Comment:
Why isn't `ReplaceLineEndings` enough for all cases?

Reply:
ReplaceLineEndings isn't available on .NET Framework, it's an API added in .NET 6.